doc/readme/INSTALL.md

```diff
@@ -297,7 +297,13 @@ You will need to adjust all path names to the Windows style, e.g.
 
 You need to specify a different generator in cmake (to match your IDE), e.g.
 
-    cmake -G"Visual Studio 14 2015 Win64" -DCMAKE_BUILD_TYPE=Release -DBUILD_META_EXAMPLES=OFF -DENABLE_TESTING=ON ..
+    cmake -G"Visual Studio 14 2015 Win64" -DCMAKE_BUILD_TYPE=Release -DBUILD_META_EXAMPLES=OFF -DENABLE_TESTING=ON ..  
+The above ```cmake``` has the following arguments:    
+1. ```Visual Studio 14 2015 Win64``` specifies the target platform to be x64.   
+2. ```DCMAKE_BUILD_TYPE=Release``` specifies a build type and asks compiler to perform optimization and omit debug information.     
```

Comment:
-DCMAKE_BUILD_TYPE=Release

Reply:
@vigsterkr Thank you for all the suggestions and reviews. I have made the suggested changes. Let me know if any other changes are required.